docs/cep/reference/extensions/execution/sentiment.md

```diff
@@ -11,7 +12,7 @@ This extension performs sentiment analysis using Afinn Wordlist-based approach.
    
 ## getRate
 
-This provides the sentiment value for a given string as per the AFINN word list.
+This provides the sentiment value for a given string as per the AFINN word list. The sentiment value ranges from -5 to +5.
```

Comment:
Didn't we just say this on line 2? I suggest removing it in one place or the other.